lighthouse-cli/test/smokehouse/lighthouse-runners/cli.js

```diff
@@ -31,9 +32,10 @@ const ChildProcessError = require('../lib/child-process-error.js');
  * @return {Promise<{lhr: LH.Result, artifacts: LH.Artifacts, log: string}>}
  */
 async function runLighthouse(url, configJson, testRunnerOptions = {}) {
-  const tmpPath = await fs.mkdtemp(`${os.tmpdir()}/smokehouse-`);
-
   const {isDebug} = testRunnerOptions;
+  const tmpDir = isDebug ? `${LH_ROOT}/.tmp/smokehouse` : os.tmpdir();
+  await fs.mkdir(tmpDir, {recursive: true});
+  const tmpPath = await fs.mkdtemp(`${tmpDir}/smokehouse-`);
   return internalRun(url, tmpPath, configJson, testRunnerOptions)
```

Comment:
drive-by improvement: if debug mode, save the things in a readily-accessible folder inside `.tmp`.

Reply:
> drive-by improvement: if debug mode, save the things in a readily-accessible folder inside .tmp.

good idea, though we delete the directory if `!isDebug`, so might as well just always put it in `.tmp` (I think we just hadn't settled on `.tmp` way back when and it felt gross just saving in root or whatever)